rewrite-maven/src/main/java/org/openrewrite/maven/tree/MavenRepository.java

```diff
@@ -116,6 +118,15 @@ public static class Builder {
         private Builder() {
         }
 
+        @VisibleForTesting
+        @Nullable
+        public static String getEnvironmentPropertyName(@Nullable String rawProperty) {
+            if (rawProperty == null){
+                return null;
+            }
+            return rawProperty.replace("${env.", "").replace("}", "");
+        }
```

Comment:
This is the best I was able to come up with to be able to test this method for the positive scenario. 

I did not find any test in this source code that works with environment properties and it is currently enabled